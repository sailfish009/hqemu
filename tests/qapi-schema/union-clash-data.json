# Union branch 'data'
# FIXME: this parses, but then fails to compile due to a duplicate 'data'
# (one from the branch name, another as a filler to avoid an empty union).
# we should either detect the collision at parse time, or change the
# generated struct to allow this to compile.
{ 'union': 'TestUnion',
  'data': { 'data': 'int' } }
