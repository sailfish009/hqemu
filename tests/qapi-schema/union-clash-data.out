object :empty
object :obj-int-wrapper
    member data: int optional=False
object TestUnion
    case data: :obj-int-wrapper
enum TestUnionKind ['data']
